index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>David's Portfolio - Mobile</title>
+</head>
+<body>
+    <section class="works">
+        <div class="grid-container">
+            <div class="card card1">
+                <img class="screenshot" src="./pics/homepage.png" alt="">
+                
+                <h1 class="main-heading"> Tonic </h1>
+                
```

Comment:
- [ ] Please, consider that the `h1` tag could be used one time on your webpage. All other headings should be `hx`, aka h2, h3, h4, etc.
- [ ] Please, fix it for all extra `h1` tags in your code.

Reply:
Well noted thanks. It has been implemented.